src/main/java/com/minecolonies/coremod/commands/ChangeColonyOwnerCommand.java

```diff
@@ -62,37 +62,25 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
         }
 
         int colonyId = getIthArgument(args, 0, -1);
-
         if(colonyId == -1)
         {
             final String playerName = args[0];
-            if(playerName == null || playerName.isEmpty())
-            {
-                sender.getCommandSenderEntity().addChatMessage(new TextComponentString(NO_PLAYER));
-                return;
-            }
 
-            final EntityPlayer player = sender.getEntityWorld().getPlayerEntityByName(playerName);
-            if(player == null)
+            if(playerName == null || playerName.isEmpty())
             {
                 sender.getCommandSenderEntity().addChatMessage(new TextComponentString(NO_PLAYER));
                 return;
             }
-
+            EntityPlayer player = sender.getEntityWorld().getPlayerEntityByName(playerName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)